config/application.rb

```diff
@@ -5,6 +5,7 @@
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
+Dotenv.load('.env')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.